src/globalvars.cpp

```diff
@@ -102,7 +102,7 @@ static std::map<GlobalVar, int> find_btf_var_offsets(
       vars_and_offsets[*global_var] = member->offset;
   }
 
-  for (const auto [global_var, offset] : vars_and_offsets) {
+  for (const auto &[global_var, offset] : vars_and_offsets) {
```

Comment:
This is the type of vars_and_offsets `std::map<GlobalVar, int> vars_and_offsets;` (GlobalVar is an enum) so I don't think we lose anything by copying these two but if it makes the linter happy...

Reply:
Yep, you're right, `note: use reference type to prevent copying` is not important. We just close this PR?

Reply:
Let's wait to see if anyone else has a thought. Reducing noise from the linter might be justification enough but also using a const ref is more correct if the underlying type were to ever change.